modin/pandas/base.py

```diff
@@ -3165,17 +3197,21 @@ def __get_groups(
     @property
     def groups(self):
         return self._query_compiler.default_to_pandas(
-            lambda df: pandas.DataFrame.resample(df, *self.resample_args).groups
+            lambda df: pandas.DataFrame.resample(
+                df, *list(self.resample_args.values())
```

Comment:
```suggestion
                df, *self.resample_args.values()
```

An extra list construction can be excluded

Reply:
This construction list was deleted.